types/react-loadable/webpack.d.ts

```diff
@@ -0,0 +1,23 @@
+import * as webpack from 'webpack';
```

Comment:
webpack is a commonjs module; import it like a commonjs module.

`import webpack = require('webpack')`